app/views/shared/_application_information.html.erb

```diff
@@ -66,8 +66,6 @@
       </table>
     </div>
   </div>
-    <% if @planning_application.policy_evaluation %>
-      <%= render "planning_applications/policy_consideration_list", policy_considerations: @planning_application.policy_evaluation.policy_considerations %>
-    <% end %>
+    <%= render "planning_applications/policy_consideration_list" %>
```

Comment:
do we no longer need the if statement here? Are proposal details now compulsory to a successful api call?

Reply:
It is not required, but if the proposal_details are nil, then the .each in the partial will just not render anything.